pkg/registry/event/rest.go

```diff
@@ -45,7 +46,8 @@ func NewStorage(registry generic.Registry) *REST {
 func (rs *REST) Create(ctx api.Context, obj runtime.Object) (runtime.Object, error) {
 	event, ok := obj.(*api.Event)
 	if !ok {
-		return nil, fmt.Errorf("invalid object type")
+		glog.V(2).Infof("Invalid object type. Was expecting an object of type api.Event: %v", obj)
```

Comment:
I would vote against this log message. It's going to be very hard to associate with a request. Instead, make the error you return contain this information? e.g. ```fmt.Errorf("expected *api.Event, but got %T", obj)```